src/minType.js

```diff
@@ -299,6 +299,22 @@ function minType (sup, sub) {
     return consistencyCheck(computed)
   }
 
+  if (subType === 'union' && superType !== 'union') {
+    let anyOf = sub.anyOf
+    let unionResult = sub
```

Comment:
Is it safe to modify the given object? Elsewhere in this function we create a new object.